src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -44,7 +44,7 @@ public Command parseCommand(String userInput) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE));
         }
 
-        final String commandWord = matcher.group("commandWord");
+        final String commandWord = matcher.group("commandWord").toLowerCase();
```

Comment:
Nice! But we need some testings for this

Reply:
Would the UI look uglier because of the random word in message_usage?

Reply:
Thank you for the feedback, ive removed the random case msg in msg_usage so that the UI can look cleaner.